nixos/modules/services/audio/navidrome.nix

```diff
@@ -51,6 +51,7 @@ in
               description = "Enable anonymous usage data collection, see <https://www.navidrome.org/docs/getting-started/insights/> for details.";
               type = bool;
             };
+            Plugins.Enabled = mkEnableOption "Enable running plugins";
```

Comment:
Would it be possible to add `plugins` to `passthru` in the package, so that we could enable this dynamically? I don't think it even needs to be declared as an option at that point, you could just set the following in the `config`

```nix
  services.navidrome.settings.Plugins.Enabled = lib.mkDefault (cfg.package.plugins != [ ]);
```